src/aio/disk_engine.cpp

```diff
@@ -126,9 +126,9 @@ aio_task *disk_file::on_write_completed(aio_task *wk, void *ctx, error_code err,
         if (err == ERR_OK) {
             size_t this_size = (size_t)wk->get_aio_context()->buffer_size;
             dassert(size >= this_size,
```

Comment:
Better to use `dcheck_ge` to simplify code.

Reply:
done